tpl/ase_gauss_ts_end.tpl.py

```diff
@@ -33,11 +33,7 @@
 mol = Atoms(symbols = atom, positions = geom)
 mol.set_calculator(calc)
 try:
-    db2 = connect('{working_dir}/geoms2_end.db')
-    db2.write(mol, name=label)
     e = mol.get_potential_energy() # use the Gaussian optimizer
-    db3 = connect('{working_dir}/geoms3_end.db')
-    db3.write(mol, name=label)
```

Comment:
same comment as in the case of ase_gauss_ts_search.tpl.py

Reply:
also removed.